src/content/language/libraries.md

```diff
@@ -89,29 +89,24 @@ import 'package:lib1/lib1.dart' show foo;
 import 'package:lib2/lib2.dart' hide foo;
 ```
 
-<a id="deferred-loading"></a>
-#### Lazily loading a library
+#### Lazily loading a library {:#lazily-loading-a-library}
 
-_Deferred loading_ (also called _lazy loading_)
+*Deferred loading* (also called *lazy loading*)
 allows a web app to load a library on demand,
 if and when the library is needed.
-Here are some cases when you might use deferred loading:
+Use deferred loading when you want to meet one or more of the following needs.
 
-* To reduce a web app's initial startup time.
-* To perform A/B testing—trying out
+* Reduce an Android or web app's initial startup time.
+* Perform A/B testing—trying out
   alternative implementations of an algorithm, for example.
-* To load rarely used functionality, such as optional screens and dialogs.
-
-:::warning
-**Only `dart compile js` supports deferred loading.**
-Flutter and the Dart VM don't support deferred loading.
-To learn more, see
-[issue #33118](https://github.com/dart-lang/sdk/issues/33118) and
-[issue #27776.](https://github.com/dart-lang/sdk/issues/27776)
-:::
+* Load rarely used functionality, such as optional screens and dialogs.
+
+The Android and JavaScript deferred loading differ.
+The Android app bundles all deferred components into the APK at compile time.
```

Comment:
My understanding is that this support for deferred loading on Android functionality is specific to Flutter, and other Dart users can't take advantage it without further customizations to the VM or their embedder.

Due to this and the fact we don't generally document Flutter special handling on the site, I'd instead recommend not mentioning Android in this discussion, and linking to the Flutter site.

Maybe something roughly like:

```
The `dart` tool doesn't support deferred loading for its other compilation targets.
If you're building a Flutter app, to learn about the platform it supports for deferred loading,
check out the flutter.dev docs on
[deferred components](https://docs.flutter.dev/perf/deferred-components).
```

That also prevents us from having to make further updates if Flutter changes that functionality or implements support for other target platforms.